### PR TITLE
Compatibility with signac 1.8

### LINKED ISSUE #132: Compatibility with signac 2.0
Running list of things that need to be changed for signac 2.0 (`next` branch)
- remove reliance on project titles
- use `job_id` instead of `jobid` everywhere

### PR DESCRIPTION
**Pull request by cbkerr**
## Description

Merge after #152 is merged.

## Motivation and Context

Resolves #132 

## Checklist:

- [x] I am familiar with the [Contributing Guidelines](https://github.com/glotzerlab/signac-dashboard/blob/master/CONTRIBUTING.md).
- [x] I agree with the terms of the [Contributor Agreement](https://github.com/glotzerlab/signac-dashboard/blob/master/ContributorAgreement.md).
- [x] The changes introduced by this pull request are covered by existing or newly introduced tests.
- [x] The [package documentation](https://github.com/glotzerlab/signac-dashboard/tree/master/doc) and [framework documentation](https://docs.signac.io/) in [signac-docs](https://github.com/glotzerlab/signac-docs) are up to date with these changes.
- [x] I have updated the [changelog](https://github.com/glotzerlab/signac-dashboard/blob/master/changelog.txt) and added any related issue and pull request numbers for future reference.

### CODE REVIEW

**Comment by bdice** [comment]
@cbkerr We can’t merge this until signac 2.0 is released, right? If not, I’d like to keep this in draft until then.

File: signac_dashboard/views.py [hunk 1]
```diff
@@ -45,16 +47,19 @@ def jobs_list(dashboard):
     if not jobs:
         flash("No jobs found.", "warning")
     g.jobs = dashboard._get_job_details(g.pagination.paginate(jobs))
-    project_title = dashboard.project.config.get("project", None)
-    g.title = f"{project_title}: Jobs" if project_title else "Jobs"
+    project_path = dashboard.project.config["project_dir"]
+    _, project_folder = split(project_path)
```
Comment (bdice) [inline signac_dashboard/views.py:51]: Do you want `os.path.basename` instead?

Comment (bdice) [reply signac_dashboard/views.py:51]: Also I’d prefer to call it a directory instead of a folder.

Comment (cbkerr) [reply signac_dashboard/views.py:51]: Also open for discussion is what the title should actually say now!

**Comment by bdice** [comment]
@cbkerr Maybe we can refactor this to focus on 1.8+ compatibility and require `signac>=1.8.0` like this PR: https://github.com/glotzerlab/signac-flow/pull/693

File: signac_dashboard/views.py (hunk 1 above)
Comment (cbkerr) [reply signac_dashboard/views.py:51]: Discussed in dev meeting this week. We'll just say "signac dashboard" for now

File: signac_dashboard/dashboard.py [hunk 2]
```diff
@@ -382,9 +378,6 @@ def _job_search(self, query):
                     query = shlex.split(query)
                     f = signac.contrib.filterparse.parse_filter_arg(query)
                     flash(f"Search string interpreted as '{json.dumps(f)}'.")
-            if querytype == "document":
-                jobs = self.project.find_jobs(doc_filter=f)
-            else:
                 jobs = self.project.find_jobs(filter=f)
```
Comment (bdice) [inline signac_dashboard/dashboard.py:381]: Remove an indentation level.
```suggestion
            jobs = self.project.find_jobs(filter=f)
```

File: signac_dashboard/views.py [hunk 3]
```diff
@@ -45,16 +45,14 @@ def jobs_list(dashboard):
     if not jobs:
         flash("No jobs found.", "warning")
     g.jobs = dashboard._get_job_details(g.pagination.paginate(jobs))
-    project_title = dashboard.project.config.get("project", None)
-    g.title = f"{project_title}: Jobs" if project_title else "Jobs"
+    g.title = f"signac-dashboard: jobs"
```
Comment (bdice) [inline signac_dashboard/views.py:48]:
```suggestion
    g.title = "signac-dashboard: Jobs"
```

File: signac_dashboard/views.py [hunk 4]
```diff
@@ -45,16 +45,14 @@ def jobs_list(dashboard):
...
     g.subtitle = g.pagination.item_counts()
     return dashboard._render_job_view(default_view="list")
 
 
 def project_info(dashboard):
     g.project = dashboard.project
-    project_title = dashboard.project.config["project"]
-    g.title = str(project_title)
+    g.title = f"signac-dashboard"
```
Comment (bdice) [inline signac_dashboard/views.py:55]:
```suggestion
    g.title = "signac-dashboard"
```

File: tests/test_dashboard.py [hunk 5]
```diff
@@ -54,17 +53,17 @@ def test_invalid_token(self):
...
         rv = self.test_client.get("/project/", follow_redirects=True)
         response = str(rv.get_data())
-        assert "dashboard-test-project" in response
+        assert "signac-dashboard" in response
 
     def test_get_jobs(self):
         rv = self.test_client.get("/jobs/", follow_redirects=True)
         response = str(rv.get_data())
-        assert "dashboard-test-project" in response
+        assert "signac-dashboard: jobs" in response
```
Comment (bdice) [inline tests/test_dashboard.py:61]:
```suggestion
        assert "signac-dashboard: Jobs" in response
```

**Review by bdice — APPROVED** [review]
This looks pretty close to me. A few suggestions, then feel free to merge when you are ready.

Please remember to update the changelog.

**Review by bdice — APPROVED** [review]
Local testing looks fine now. I'll merge this when CI passes.

**Pull request merged**